pkgs/by-name/sp/spooftooph/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchurl
+, bluez
+, ncurses
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "spooftooph";
+  version = "0.5.2";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/spooftooph/spooftooph-${finalAttrs.version}/spooftooph-${finalAttrs.version}.tar.gz";
+    hash = "sha256-JH5+fHpe83NJV9AR5MXKnrwaTqz4s2BGAcczbddVNHw=";
+  };
+
+  buildInputs = [
+    bluez
+    ncurses
+  ];
+
+  postPatch = ''
+    substituteInPlace makefile \
+      --replace-fail /usr/bin $out/bin
+  '';
```

I'm pretty sure this can be done with `makeFlags` somehow, but for some reason my machine insists on building world from scratch while testing this, so I can't say for sure exactly how to do it at the moment.

Could you try something like this?

```suggestion
  makeFlags = [ "BIN=$(out)/bin" ];
```

---

It works. Fixed.